move-to-sdk/app/io/sphere/sdk/play/metrics/MetricAction.java

```diff
@@ -44,6 +45,7 @@ private void logRequestDataOnComplete(final Http.Context ctx, final List<ReportR
     }
 
     private Integer calculateTotalSize(final List<ReportRawData> rawDatas) {
-        return rawDatas.stream().map(elem -> elem.getHttpResponse().getResponseBody().map(b -> b.length).orElse(0)).reduce(0, (a, b) -> a + b);
+        return rawDatas.stream().map(elem -> Optional.ofNullable(elem.getHttpResponse().getResponseBody())
+                .map(b -> b.length).orElse(0)).reduce(0, (a, b) -> a + b);
```

Comment:
I created a ticket for it: https://github.com/sphereio/sphere-sunrise/issues/162